swin_transformer/slurm_train.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+set -x
+
+REPO=$1
+PARTITION=$2
+JOB_NAME=$3
+CONFIG=$4
+WORK_DIR=$5
+GPUS=${GPUS:-8}
+GPUS_PER_NODE=${GPUS_PER_NODE:-8}
+CPUS_PER_TASK=${CPUS_PER_TASK:-5}
+SRUN_ARGS=${SRUN_ARGS:-"-x SH-IDC1-10-198-4-[92,94]"}
```

Comment:
Should not include information about the developer's environment. Besides, it is recommended to launch training/testing with mim.